app/controllers/admin_request_controller.rb

```diff
@@ -87,22 +87,23 @@ def destroy
 
```

Comment:
Commit message for 8b5f874579dd5d4ea92e1b71228cdece84be7a39: did you mean s/explaining/example/ ?

Reply:
Also minor nit-pick commit also changes the hash styling

Reply:
> did you mean s/explaining/example/ ?

Nerp, it's a Thing – https://wiki.c2.com/?IntroduceExplainingVariable

Reply:
fixed